src/cli_config.rs

```diff
@@ -97,13 +108,15 @@ pub struct CliConfig {
     /// List of bootstrap nodes.
     pub bootstrap_nodes: Vec<BootstrapNode>,
     /// Number of threads for execution.
-    pub threads_config: ThreadsConfig,
+    pub threads_config: Option<ThreadsConfig>,
```

Comment:
Why make it optional?

Reply:
Making it optional gives cleaner code, because getting from config is saved as Option.
It shows that `thread` argument is given or not as a cli args.

Reply:
But default value is displayed in the help message. We can check if it was specified as an argument using `occurrences_of` method.

Reply:
I don't understand why we should use `Option`. @NamsooCho what are your thoughts and reasons about that?

Reply:
@kpp Why should not use Option?

Reply:
I suppose @NamsooCho changed it to optional because it conflicted with config arg. But it seems if we use a subcommand we don't have to touch existing arguments parsing, and we can easily check if we should use config file or args.